### DISCUSSION TITLE
👀 What's new in GitHub Mobile?

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#135039] [Mobile]
![Screenshot 2024-08-08 at 11 07 05 AM](https://github.com/user-attachments/assets/158a85f6-d68a-4dc5-b53f-152245e4677f)

In July, GitHub Mobile introduced three major improvements:

*   [App Lock!](https://github.blog/changelog/2024-08-06-app-lock-via-face-id-or-biometrics/) ➡️ Securely unlock the GitHub app with just a glance. Enable App Lock in Settings to use FaceID, TouchID or pass code to protect your information in the GitHub app.
*   [A smarter Copilot Chat!](https://github.blog/changelog/2024-07-31-github-copilot-chat-and-pull-request-summaries-are-now-powered-by-gpt-4o/)  ➡️  It knows where you are in the app. Ask Copilot about the file or repository you’re viewing to try it out.
*   [Workflow Dispatching!](https://github.blog/changelog/2024-07-30-run-workflows-set-as-workflow_dispatch-manually/)  ➡️  Kick off new Actions on the go from the list of workflow runs for a given workflow.

We've released several new fixes and features for both iOS and Android apps. You can explore these updates in our [changelog post](https://github.blog/changelog/2024-08-07-whats-new-in-mobile-july-update/#as-well-as-several-other-fixes-and-features-to-both-ios-and-android-apps). Have you had a chance to try them out? We’d love to hear your feedback—please share your thoughts in the comments below! 👇

**Comment by UjjwalSaini07** [comment]
It invites engagement and interaction, which is crucial for building a community around your content. The pointing-down emoji adds a friendly and informal tone, making the request feel more personal and approachable. Overall, it's a great way to encourage feedback and discussion, which can lead to more meaningful interactions and insights from your audience.

**Comment by shahbazm788** [comment]
You can now update your Profile status directly from GitHub Mobile on Android. On both iOS and Android, you will find improvements in large accessibility sizes, better content descriptions and keyboard navigation, with particular focus on the “Request Reviewers” and “Merge Options” screens
